### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by BasitKorede**

### CODE REVIEW

**Comment by BasitKorede** [comment]
This is a landing page.

**Comment by BasitKorede** [comment]
#### this website is a landing page, it a free camp code assignment.
##### thing to see on this branch:
* header.
* navigation-bar.
* navigation-links.
* the company logo.
* an email input field .
* an embedded video.

File: css/style.css [hunk 1]
```diff
@@ -27,6 +30,7 @@ body{
   text-align: center;
   flex-grow: 1;
   font-weight: bolder;
+  font-size: 150%;
 }
 .nav-link:hover{
   opacity: 0.8;
```
Comment (adejam) [inline css/style.css:36]: we should adopt the hex color or RGB color format when using colors.

File: css/style.css [hunk 2]
```diff
@@ -38,14 +42,26 @@ body{
   margin-left: auto;
   margin-right: auto;
   margin-top:20px;
+  background-color: #f5f5dc;
+  padding: 5px;
 }
 #tips{
   margin-top:px;
 }
```
Comment (adejam) [inline css/style.css:50]: we should add value to the margin here

**Review by adejam — APPROVED** [review]
LGTM!

Let's make the following changes before merging

**Pull request merged**